### PR TITLE
Debugging burundi

### PR DESCRIPTION
**Pull request by ghalter**

### CODE REVIEW

**Comment by mastaia** [comment]
@ghalter could you quickly double check if everything is solid :)?

File: pipeline/indirect/indirect.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import pandas as pd
 import pycountry
 from climada_petals.engine import SupplyChain
+from exposures.utils import root_dir
+project_root = root_dir()
```
Comment (ghalter) [inline pipeline/indirect/indirect.py:6]: No reason to put this here I guess, just use it in your function.

File: pipeline/indirect/indirect.py [hunk 2]
```diff
@@ -90,10 +119,19 @@ def get_secs_shock(supchain: SupplyChain, country_iso3alpha, impacted_secs, n_to
...
-        row_fract_per_county = 1 / (n_total - (len(set(r[0] for r in supchain.mriot.x.axes[0])) - 1))
-        return row_fract_per_county * supchain.secs_shock.loc[:, (impacted_secs, "ROW")]
+
+        #row_fract_per_county = 1 / (n_total - (len(set(r[0] for r in supchain.mriot.x.axes[0])) - 1)) #simplest version TODO to be deleted
+
+        #Using the GDP values to create a factor
+        ROW_GDP = get_gdp_modifier(supchain)
+        row_fract_per_county =  ROW_GDP.loc[ROW_GDP['Country Code'] == country_iso3alpha, 'Normalized_GDP'].values[0]
+        if np.isnan(row_fract_per_county):
+            row_fract_per_county = 0
```
Comment (ghalter) [inline pipeline/indirect/indirect.py:132]: I would move these lines into the get_gpd_modifier function too.

**Review by ghalter — COMMENTED** [review]
Thanks @mastaia, some stuff to clean, the approach looks sounds to me

File: pipeline/indirect/indirect.py (hunk 1 above)
Comment (mastaia) [reply pipeline/indirect/indirect.py:6]: yes, moved it to the function

**Pull request merged**